conventions/documentation/create.md

```diff
@@ -45,7 +91,7 @@ parent: Conventions
 
 ---
 
-**Create from button (Free switching between Form & YAML):**
+### Creation via free switching between form & YAML:
 
 *Some resource types can be created with both a form and YAML editor. The user is able to freely switch between the two editors and changes are preserved across them. After the resource is created, the user is automatically navigated to the created resource.*
```

Comment:
The images make it look like the user is redirected to the list view after creation.

Reply:
Not sure I follow. The image of the Routes list view with the pointer clicking "Create Route"? That image was intended to set up the next 3 images showing the form, then YAML, then details view.

Reply:
Ah sorry, should have explained better. The last image (after the user clicks Create on the YAML view) shows the KieApps list view rather than the KieApp resource's details view. As is, the text "After the resource is created, the user is automatically navigated to the created resource." and image don't align!

Reply:
Ha thanks! I described the convention then showed an example in the console that doesn't follow it, whoops. I will update this.